app/views/applications/applications.scala.html

```diff
@@ -38,19 +38,24 @@ <h1>Applications</h1>
             action="@routes.ApplicationController.applicationsPage().url"
             method="get">
 
-                <div class="grid-layout grid-layout--stacked push--sides js-visible">
+                <div class="form-group grid-layout grid-layout--stacked push--sides js-visible">
 
-                    <div class="grid-layout__column--1-4">
-                        <label class="bold centered-content hard--left">
+                    <div class="grid-layout__column">
+                        <label class="bold centered-content hard--left label--full-length">
                             <span class="label__text bold">Search</span>
-                            <input class="form-control-7-8 input--cleared input-select--large"
-                            data-datatable-custom-filter="applications-table"
-                            name="search"
-                            type="text"
-                            placeholder="Name" />
+
+                            <input  class="form-control input--cleared input-select--large"
```

Comment:
might just be me but this input doesn't look tall enough for its containing text - probably some design guidelines thing though

Reply:
Nah I was thinking the same thing... I left it as I found it though. Just in case.